androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt

```diff
@@ -105,6 +105,8 @@ class ComposePrototypeUITest : BaseTests() {
     private val helloSession1 = "Session 1 - Authenticated message."
     private val helloSession2 = "Session 2 - Unauthenticated message."
     private val helloResumeSession1 = "Session 1 - Attempt to resume after step-up."
+    private val implicitFlowMessage = "Implicit flow authenticated message."
```

Comment:
We do not run this test for almost 2 years (they will have to be removed one day). This file should not be touched